File: src/main/java/love/simbot/example/Log_settler.java
package love.simbot.example;
import java.io.BufferedWriter;
import java.io.File;
import java.io.FileWriter;
import java.io.IOException;
import java.text.SimpleDateFormat;
import java.util.Date;
public class Log_settler {
    public static void settle() throws IOException {
        Date date=new Date();
        SimpleDateFormat formatter = new SimpleDateFormat("yyyyMMdd");
        String date1= formatter.format(date);
        File dir = new File("./cache/logs");
        dir.mkdirs();
        File log= new File ("./cache/logs/"+date1+".log");
        if (!log.exists()){
            log.createNewFile();
        }
    }
    public static void writelog(String content) throws IOException {
        Date date2=new Date();
        SimpleDateFormat formatter2 = new SimpleDateFormat("yyyyMMdd");
        String date3= formatter2.format(date2);
        File log1= new File ("./cache/logs/"+date3+".log");
        BufferedWriter output= new BufferedWriter(new FileWriter(log1));
        output.write(content+"\n");
        output.close();
    }
}

File: src/main/java/love/simbot/example/Log_settler.kt
package love.simbot.example

import java.io.File
import java.time.ZonedDateTime
import java.time.format.DateTimeFormatter
import java.util.logging.*

class Log_settler {
    val mylog: Logger
        get() {
            val time = ZonedDateTime.now()
            val date1 = time.format(DateTimeFormatter.ofPattern("yyyyMMdd"))
            val logger = Logger.getLogger(date1)
            logger.level = Level.ALL
            for (h in logger.handlers) {
                h.close()
            }
            try {
                val fileHandler = FileHandler("./cache/logs/$date1.log", true)
                fileHandler.formatter = myFormat()
                logger.addHandler(fileHandler)
            } catch (e: Exception) {
                e.printStackTrace()
            }
            return logger
        }

    companion object {
        @JvmStatic
        fun settle() {
            val file = File("./cache/logs")
            file.mkdirs()
        }

        @JvmStatic
        fun writelog(content: String?) {
            val lt = Log_settler()
            val log = lt.mylog
            log.info(content)
        }
    }
}

internal class myFormat : Formatter() {
    override fun format(record: LogRecord): String {
        val zdf = ZonedDateTime.now()
        val sDate = zdf.format(DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss"))
        return """
            [$sDate]: ${record.message}
            
            """.trimIndent()
    }
}